### PR TITLE
Make model use more visible

### PR DESCRIPTION
**Pull request by osanseviero**

### CODE REVIEW

File: src/huggingface_hub/templates/modelcard_template.md [hunk 1]
```diff
@@ -32,6 +32,12 @@
 - **Paper [optional]:** {{ paper | default("[More Information Needed]", true)}}
 - **Demo [optional]:** {{ demo | default("[More Information Needed]", true)}}
 
+## How to Get Started with the Model
+
+Use the code below to get started with the model.
+
+{{ get_started_code | default("[More Information Needed]", true)}}
+
```
Comment (EziOzoani) [inline src/huggingface_hub/templates/modelcard_template.md:40]: To keep the Uses, Bias, Risks, and Limitations as a focus, I could see the *How to Get Started* section being before *Training Details* (after *Recommendations*)

File: src/huggingface_hub/templates/modelcard_template.md [hunk 2]
```diff
@@ -32,6 +32,12 @@
 - **Paper [optional]:** {{ paper | default("[More Information Needed]", true)}}
 - **Demo [optional]:** {{ demo | default("[More Information Needed]", true)}}
 
+## How to Get Started with the Model
+
+Use the code below to get started with the model.
+
```
Comment (EziOzoani) [inline src/huggingface_hub/templates/modelcard_template.md:38]: The code within the *How to get started*  can get quite long so keeping the ***Click to expand*** would be help to condense it, especially with the proposed moving to this high in the MC

**Review by meg-huggingface — COMMENTED** [review]
Do we have evidence that the consensus from users is that this should be moved higher up?

**Comment by EziOzoani** [comment]
Oo 👀,  I’m not sure yet but possibly with the user tag feature we could get a sense of which user tags fill out certain sections, in this case the *How to get Started section*. 

One possible positive of the *How to get started* being moved up could be in the scenario where one just skips to the end to find the *How to get started section* and with it being below the *Uses, Bias, Risks, and Limitations* sections, those will be more “in view”. That positive would only be in that specific scenario and I don’t know how common that use case is

**Comment by osanseviero** [comment]
> To keep the Uses, Bias, Risks, and Limitations as a focus, I wouldn't be on-board to move the How to Get Started before them but I could see the How to Get Started section being before Training Details (after Recommendations)

Yes, I agree this would be better

**Comment by Wauplin** [comment]
@osanseviero @meg-huggingface @EziOzoani thanks for the feedback. I would suggest to merge it as it is (e.g. after the _"Use, Bias,..."_ but before _"Training"_ and without the "click to expand" section). If we have negative feedback in the future, let's revisit it but I doubt.

**Pull request merged**